snapcraft/tests/test_tour.py

```diff
@@ -51,11 +49,13 @@ class TestScaffoldExample(TestCase):
     def setUp(self):
         super().setUp()
         self.temp_dir = None
+        self.cwd = os.getcwd()
 
     def tearDown(self):
         with suppress(OSError):
             if self.temp_dir:
                 shutil.rmtree(self.temp_dir)
+        os.chdir(self.cwd)
```

Comment:
I missed this in the previous review.
We have a fixture for tmp cwd, which also works as a tempdir. Maybe you can replace your set up and tear down with that fixture, or inherit from the base test case.